code/src/java/pcgen/gui2/converter/panel/WriteDirectoryPanel.java

```diff
@@ -133,9 +113,9 @@ public boolean returnAllowed()
 	public void setupDisplay(JPanel panel, final CDOMObject pc)
 	{
 		panel.setLayout(layout);
-		JLabel label = new JLabel(
+		Component label = new JLabel(
 				"Please select the Directory where Converted files should be written: ");
-		JButton button = new JButton("Browse...");
+		AbstractButton button = new JButton("Browse...");
```

Comment:
in general I think the left side of the declaration should have the weakest type possible (excluding Object).